jwplayer-appletv-web-app/resources/configs/VCyJXbpY/config.json

```diff
@@ -1,11 +1,25 @@
 {
   "playlists": [
-    "K6Sl8yPJ",
-    "BXatQw7p",
-	  "E12RS6r7"
+    {
+      "playlistId": "K6Sl8yPJ",
+      "autoAdvance": true,
```

Comment:
@RikHeijdens I'm not super concerned about the lack of flexibility here.  Let's get a good understanding of why a publisher might want to set playlist-specific behavior, and then refactor accordingly.  For now, I think simplicity should be the higher priority.

Even if we want to store more information about playlists, we can keep that in a globally accessible lookup, and use the string IDs to retrieve the relevant details.